cmd/solana_exporter/slots_test.go

```diff
@@ -91,26 +92,41 @@ func TestSolanaCollector_WatchSlots_Static(t *testing.T) {
 	leaderSlotsTotal.Reset()
 	leaderSlotsByEpoch.Reset()
 
-	collector := createSolanaCollector(&staticRPCClient{}, 100*time.Millisecond, identities, []string{})
+	collector := createSolanaCollector(&staticRPCClient{}, 100*time.Millisecond, identities, []string{}, votekeys)
 	watcher := NewCollectorSlotWatcher(collector)
 	prometheus.NewPedanticRegistry().MustRegister(collector)
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 	go watcher.WatchSlots(ctx, collector.slotPace)
+
+	// make sure inflation rewards are collected:
+	err := watcher.fetchAndEmitInflationRewards(ctx, staticEpochInfo.Epoch)
+	assert.NoError(t, err)
 	time.Sleep(1 * time.Second)
 
 	firstSlot, lastSlot := getEpochBounds(&staticEpochInfo)
-	tests := []struct {
+	type testCase struct {
 		expectedValue float64
 		metric        prometheus.Gauge
-	}{
+	}
```

Comment:
I think anonymous structs are nicer for this sort of thing, but it is really just a style debate. They're both fine.